lib/player.rb

```diff
@@ -0,0 +1,24 @@
+class Player
+  def initialize(player_one, player_one_sym, player_two_sym)
+    @player_one = player_one
+    @player_one_sym = player_one_sym
+    @player_two_sym = player_two_sym
+  end
+
+  def sym_check(current_player)
+    current_sym = if current_player == @player_one
+                    @player_two_sym
+                  else
+                    @player_one_sym
+                  end
+    current_sym
```

Comment:
Are you familiar with the [ternary operator](https://www.rubyguides.com/2019/10/ruby-ternary-operator/)?? It is an amazing method, not exclusive of ruby, that can make your code so much cleaner. **Tip**: you don't need to return `current_sym` on line 14 since ruby will do it anyway on line 9 (implicit return).